src/Datadog.Trace.ClrProfiler.Native/dd_profiler_constants.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+#include <string>
+
+#include "environment_variables.h"
+#include "logging.h"
+
+namespace trace {
+
+inline WSTRING env_vars_to_display[]{
+    environment::tracing_enabled,
+    environment::debug_enabled,
+    environment::profiler_home_path,
+    environment::integrations_path,
+    environment::include_process_names,
+    environment::exclude_process_names,
+    environment::agent_host,
+    environment::agent_port,
+    environment::env,
+    environment::service_name,
+    environment::service_version,
+    environment::disabled_integrations,
+    environment::clr_disable_optimizations,
+    environment::domain_neutral_instrumentation,
+    environment::azure_app_services,
+    environment::azure_app_services_app_pool_id,
+    environment::azure_app_services_cli_telemetry_profile_value};
+
+ inline WSTRING skip_assemblies_that_begin_with[]{
```

Comment:
Nice, but let's call this to something like `skip_assembly_prefixes` to avoid the preposition.